src/porepy/numerics/vem/mass_matrix.py

```diff
@@ -0,0 +1,327 @@
+"""
+Mass matrix classes for a discretization of a L2-mass bilinear form with constant test
+and trial functions for mixed methods (e.g. RT0, MVEM).
+
+The discretization takes into account cell volumes, porosity, time step and aperture,
+so that the mass matrix (shape (g.num_faces + g.num_cells)^2) has the following diagonal
+for the cell dof:
+g.cell_volumes * porosity * aperture / deltaT
```

Comment:
deltaT is out, please update comment (and in the file where you copied the documentation from.

Reply:
thanks, done